.github/workflows/test.yml

```diff
@@ -166,12 +166,35 @@ jobs:
             result/bin/crio-status
             result/bin/pinns
 
+  build-static-ppc64le:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v20
+        with:
+          install_url: https://releases.nixos.org/nix/nix-${{ env.NIX_VERSION }}/install
+      - uses: cachix/cachix-action@v12
+        with:
+          name: cri-o-static
```

Comment:
The cache should have a ppc64le suffix, otherwise we will override it.

Reply:
Hi @saschagrunert,
Renamed `cri-o-static` to `cri-o-static-ppc64le` as suggested.